pages/jobs/[id]/index.jsx

```diff
@@ -0,0 +1,83 @@
+import { ArrowBackIcon } from "@chakra-ui/icons";
+import { Box, Button, Link } from "@chakra-ui/react";
+import Head from "next/head";
+import { useRouter } from "next/router";
+import { useEffect, useState } from "react";
+
+import { supabase } from "@/utils/lib/supabase";
+
+import JobPostDescription from "./JobPostDescription";
+import JobPostHeader from "./JobPostHeader";
+import JobPostStats from "./JobPostStats";
+
+export default function BrowseJobPage() {
+  const router = useRouter();
+  const { id } = router.query;
+
+  const [job, setJob] = useState(null);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
```

Comment:
It can! Learnt a new thing today. Thank you for the suggestion @danhannigan

Reply:
Rad! Yeah that's a big appeal of Next is their data-fetching, and basically, this will statically render it on the Netlify site so we don't have to worry about loading and stuff! Super fast — I need to swap the data fetch on the main job index page to do this, too. The only downer is that it requires a rebuild on the server to have the latest content but the jobs already need manual approval, so, I can just kick off a build while I'm at it too.